package.json

```diff
@@ -1,7 +1,19 @@
 {
-  "name": "node-irrelon-router",
-  "version": "0.1.0",
+  "name": "node-akxe-router",
+  "version": "0.0.1",
```

Comment:
Should not reverse the version numbers unless you are starting a complete fork that will never get merged back in.